deployments/kai-scheduler/values.yaml

```diff
@@ -6,6 +6,7 @@ global:
   securityContext: {}
   imagePullSecrets: []
   gpuSharing: false
+  cdi: false
```

Comment:
I'm considering whether this should reside in the global section or be scoped directly within binder. The rationale for placing it in global is that it's an environment-level parameter that could potentially impact multiple services. However, at present, it only affects binder, which may justify keeping it localized there. What are your thoughts?

Reply:
I agree!